### ISSUE TITLE
fix: susd

### ISSUE THREAD

**Issue by FoxTheSin** [yearn/yearn-finance-v3#683]
**Describe the bug**
when sUSD in wallet (and potentially other tokens?) the deposit modal doesn't appear when clicking deposit button.

**To Reproduce**
1. Go to portfolio with susd
2. click on deposit next to susd token

**Expected behavior**
The deposit modal should appear and token should be zappable into vault (or, if unzapable, an error message should pop up in modal that states that token can't be zap'd)

**Screenshots**
normal
![image](https://user-images.githubusercontent.com/79029523/169094513-cc7f89d6-2a1e-4f20-a497-1112961874a9.png)

susd
![image](https://user-images.githubusercontent.com/79029523/169094700-8006caec-6164-4b7c-a1fd-5f5eaa2000d5.png)

**Environment Information**

**Additional context**
susd contract: 0x57Ab1ec28D129707052df4dF418D58a2D46d5f51

**Comment by xgambitox** [comment]
Yeah. First step is to determine if sUSD is supported by zapper for zapping in into vaults. If its not supported, then we should not be displaying it at all in the portfolio view, since we want to display only actionable tokens within yearn app.

**Comment by xgambitox** [comment]
Maybe there was an sUSD vault that is now deprecated, and that is why sUSD is treated as a valid underlying token for direct deposits?
